### DISCUSSION TITLE
MeasureToolsUiItemsProvider

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#5597] [Q&A]
The toolbar group created by using the new ```MeasureToolsUiItemsProvider``` gives several options that are not available in ```CoreTools.measureToolGroup``` which I am using.  The way ```MeasureToolsUiItemsProvider``` is implemented, I am afraid I am not able to use as is.

```MeasureToolsUiItemsProvider```
![image](https://github.com/iTwin/itwinjs-core/assets/83007711/508c486b-833e-462f-9c1a-9b4817471f30)

```CoreTools.measureToolGroup```
![image](https://github.com/iTwin/itwinjs-core/assets/83007711/2264b937-6b1e-4d5e-9fb0-0e4d54e24f11)

Is ```CoreTools.measureToolGroup``` meant to be different from ```MeasureToolsUiItemsProvider```.  If so, can I get list of tools being used by ```MeasureToolsUiItemsProvider``` so that I can implement this menu so that it is consistent with my UI.

Thanks,

Ovais

**Comment by aruniverse** [comment]
Here's the source code for [MeasureToolsUiItemsProvider](https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx)
This is just an example of measure tools that has some more features. You can opt out of some the default tools. You could also choose not to use that package altogether, and just use the basic ones from iTwinjs-core
